airflow/providers/cncf/kubernetes/hooks/kubernetes.py

```diff
@@ -329,11 +329,18 @@ def get_custom_object(
     def get_namespace(self) -> str | None:
         """Returns the namespace that defined in the connection"""
         if self.conn_id:
-            connection = self.get_connection(self.conn_id)
-            extras = connection.extra_dejson
-            namespace = extras.get("extra__kubernetes__namespace", "default")
-            return namespace
-        return None
+            namespace = self.conn_extras.get("extra__kubernetes__namespace")
```

Comment:
Non-blocking:   I can't seem to find it now, but I thought there was recent discussion about removing the "extra__" part of these index names.  I could be completely mistaken though and I can't find it now so maybe I'm just making this up.

Reply:
yes, i'm currently getting rid of all of that........ but that's in diff PR .... so not taking that on here....

too many irons in the fire!  and no want wants to review those prs!

Reply:
Totally cool, sorry about that.  Just making sure that one didn't already get done and we were regressing here.